watch-package.js

```diff
@@ -36,11 +44,47 @@ module.exports = function watchPackage(pkgDir, exit) {
   stdin.stderr = through()
   stdin.stdout = through()
 
+  if (typeof taskName !== 'undefined' && taskName.trim() !== '') {
+    if (!pkg.scripts[taskName]) {
```

Comment:
hmm but what if taskName is undefined cause there was no param on vargs[2] than it would crash cause we can not trim on undefined. Otherwise we can do a default action after the function definition. How you like best.